Source: written by Calvin Walton <calvin.walton@kepstin.ca>
Upstream: no
Reason: Make configure check for ar, so it can find the target-prefixed one.

--- rng-tools-4/configure.ac	2012-08-02 14:56:33.000000000 -0400
+++ rng-tools-4.new/configure.ac	2015-04-14 20:58:42.768404170 -0400
@@ -30,6 +30,7 @@
 AC_PROG_CC
 AC_PROG_RANLIB
 AC_PROG_GCC_TRADITIONAL
+AM_PROG_AR
 
 dnl Checks for header files.
 dnl AC_HEADER_STDC
